__tests__/app.test.js

```diff
@@ -52,26 +53,28 @@ describe("task 4. GET /api/arcticles/:arcticle_id", () => {
       .get(`/api/articles/${article_id}`)
       .expect(200)
       .then(({ body }) => {
-        expect(body.articles).toEqual({
-          author: expect.any(String),
-          title: expect.any(String),
-          article_id: expect.any(Number),
-          body: expect.any(String),
-          topic: expect.any(String),
-          created_at: expect.any(String),
-          votes: expect.any(Number),
-        });
+        expect(body.articles).toEqual(
+          expect.objectContaining({
+            author: expect.any(String),
+            title: expect.any(String),
+            article_id: expect.any(Number),
+            body: expect.any(String),
+            topic: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+          })
+        );
       });
   });
-  test("status: 400, responds with invalid ", () => {
+  test("4. status: 400, responds with invalid id type", () => {
     return request(app)
       .get("/api/articles/PEOOHSOEPEO")
       .expect(400)
       .then(({ body }) => {
         expect(body.msg).toBe("Invalid URL");
```

Comment:
confusing error message. This would make me think that the endpoint doesn't exist. What's wrong is that the id I passed isn't valid.